hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/bucket/OMBucketCreateResponse.java

```diff
@@ -30,11 +32,16 @@
 import javax.annotation.Nullable;
 import javax.annotation.Nonnull;
 
+import static org.apache.hadoop.ozone.om.OmMetadataManagerImpl.BUCKET_TABLE;
+
 /**
  * Response for CreateBucket request.
  */
 public final class OMBucketCreateResponse extends OMClientResponse {
```

Comment:
As an alternative, you might achieve the same by adding a class annotation with table name string.

Reply:
Done.